backend/src/schema/resolvers/reports.js

```diff
@@ -183,4 +184,7 @@ export default {
       return reviewed
     },
   },
+  FILED: {
+    ...undefinedToNullResolver(['reasonDescription']),
+  },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T19:04:08Z" title="Thursday, January 23rd 2020, 8:04:08 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -183,4 +183,10 @@ export default {
       return reviewed
     },
   },
+  FILED: {
+    reasonDescription: async (parent, _params, _context, _resolveInfo) => {
+      if (typeof parent.reasonDescription !== 'undefined') return parent.reasonDescription
+      return null
+    },
+  },
```


I guess we can't use the `unfefinedToNull` resolver for `relationships`??

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-24T16:54:13Z" title="Friday, January 24th 2020, 5:54:13 pm +01:00">Jan 24, 2020</time>_

---


Thanks for the Hint! I use `undefinedToNullResolver` now …